### ISSUE TITLE
[BUG] Asset values read 0 on pairing device

### ISSUE THREAD

**Issue by realpaaao** [vultisig/vultisig-ios#1712]
**Describe the bug**
When joining a keysign, asset values read 0 on the pairing devices.

**Expected behavior**
Please display correct values on pairing devices as on initiating devices.

**Screenshots**
![Image](https://github.com/user-attachments/assets/ef9e1477-62d4-4c2f-af52-5f11367e87fa)

System:
 - Version 1.10.0

**Comment by johnnyluo** [comment]
did this one get fixed by https://github.com/vultisig/vultisig-ios/pull/1748?

**Comment by realpaaao** [comment]
@johnnyluo yes, I just tried to reproduce on 1.10.1 and looks good now. Closing
